Vorlagen/XXXValueBuilder.go

```diff
@@ -1,15 +1,11 @@
 package generator
```

Comment:
brauchts das File überhaupt noch? oder kann man die GetValues Function in den GenericCopyValueBuilder auslagern?

Reply:
bzw nur noch den verwenden, nachdem ich gerade gesehen habe, dass dort auch die function schon implementiert ist

Reply:
Das war ursprünglich nur mal als Schablone gedacht um das Gerüst für einen neuen Builder schnell zur Verfügung zu haben. Ich habs jetzt mal gelöscht.